db/models/user.js

```diff
@@ -8,15 +8,26 @@ const User = db.define('users', {
   username: Sequelize.STRING,
   first_name: Sequelize.STRING,
   last_name: Sequelize.STRING,
+  phone_number: Sequelize.STRING,
   email: {
     type: Sequelize.STRING,
+    allowNull: false,
     validate: {
       isEmail: true
     }
   },
+  // it may be worthwhile to change 'role' to an 'isAdmin' boolean
+  //    and make advocate a team instead
   role: {
     type: Sequelize.ENUM('admin', 'advocate')
   },
+  advocate_type: {
+    type: Sequelize.ENUM('Therapy', 'Legal', 'Financial', 'Job Coach', 'General')
+  },
+  team: {
+    type: Sequelize.ENUM('Dev', 'Social Media', 'Fundraising', 'Crowdsurfing'),
+    allowNull: false
```

Comment:
But what if I'm an advocate and I don't belong to a team? (I know you want to make them a team, but just saying, the way this is set up now...)